[Chronicles of Darkness] Normalization of translation keys

luizbgomide opened the pull request:
## Changes / Comments
Removing all colon from translation values, normalizing the names of translation keys, minor cosmetic fixes.

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
Yes.
- [ ] Is this a bug fix?
No.
- [ ] Does this add functional enhancements (new features or extending existing features) ?
No.
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
No.
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
No changes.
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?
Not applicable.

If you do not know English. Please leave a comment in your native language.

wing-it commented:
Please clarify, did you change the names of any translation keys? If so, any existing translations for those keys would be lost.

luizbgomide commented:
I removed colons from translation values (now it is "Name" instead of "Name:", and therefore and removed duplicates that existed before (like Krewe and Krewe-c that differ only in having a colon), replacing the key with colon with the one without it, and adding hardcoded colons where needed to avoid having duplicate keys.

The only keys removed are the duplicates, I don't think that it can affect any translations because I was the one that provided a valid translation.json last week. And even if someone started to work on a translation in this short time on the worst case scenario some label will show two colons (like "Name::") until they fix it, nothing will break.

Just a couple keys needed to be removed, but about 20 values lost a final colon to keep uniformity. The rest are cosmetic changes here and there that changes nothing.

I'm the one providing the translation to Portuguese in Crowdin, as far as I can tell there is none in other languages. I began working on the translation.json (and CofD.html), because before last week the sheet wasn't translatable (the filename was wrong), now it is all right and fixed up.

luizbgomide commented:
Ah, the translation.json was also outdated, and had keys defined more than one, so I had to generate a new one that contains all the keys. I did it so keeping the order of appearance, this is why the order is different and file looks completely changed.
